pkgs/by-name/ma/maplestory-cursor/package.nix

```
@@ -0,0 +1,53 @@
+{
+  fetchFromGitHub,
+  lib,
+  stdenvNoCC,
+
+  # build deps
+  clickgen,
+  python3Packages,
+}:
+
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "maplestory-cursor";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "Liam-Weitzel";
+    repo = "maplestory-cursor";
+    rev = "main";
```

Use tagged version or full commit hash

```suggestion
    rev = "ab043bdc2ee752141be89f6c4c4080c459d1243b";
```